pages/windows/del.md

```diff
@@ -0,0 +1,31 @@
+# del
+
+> Delete one or more files.
+
+- Delete one or more space-separated files or patterns:
+
+`del {{file_pattern}}`
+
+- Prompt for confirmation before deleting each file:
+
+`del {{file_pattern}} /p`
+
+- Force the deletion of read-only files:
+
+`del {{file_pattern}} /f`
+
+- Recursively delete files from all subdirectories:
+
+`del {{file_pattern}} /s`
+
+- Do not prompt when deleting files based on a global wildcard:
```

Comment:
What does "based on a global wildcard" mean ?

Reply:
An example would be `del *` or `del *.*` (both do the same thing).

Reply:
But that has nothing to do with the `/q` switch right ? You can do `del file.txt /q` and `del *.txt /q` and both will do the same thing. The description makes it look like the the opposite is also true - i.e. when files are deleted without a wildcard, it prompts even if you specify `/q`.

Reply:
The `/q` switch is only used to allow deletion of everything in a directory using either `*` or `*.*`. Specifying a wildcard with an extension (e.g. `*.txt`) won't prompt. You're right that `/q` won't do anything for `*.txt` or `file.txt` as it only applies when a wildcard is used that will delete all content.

Reply:
Ah I see now. Perhaps you should just specify * or *.* in brackets. Because it applies only on those cases.

Reply:
Sounds good. :+1: Would improve the readability and reduce confusion.